library/Rules/Fibonacci.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+/**
+ * @author Samuel Heinzmann <samuel.heinzman@swisscom.com>
+ */
+class Fibonacci extends AbstractRule
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($input)
+    {
+        $fib[0] = 0;
+        $fib[1] = 1;
```

Comment:
1. Avoid abbreviations, `$sequence` would be a better name, IMO.
2. Always declare a variable before you use it, so you should not use `$fib[0] = 0` until you declare `$fib = []`.

In this case, I think `$sequence = [0, 1]` would be better.